### DISCUSSION TITLE
Where are Jamulus' settings are stored on macOS?

### DISCUSSION THREAD

**Discussion by jregener62** [jamulussoftware/jamulus#1939] [Technical Support & Help] [answered]
I try to reset all settings like Profile or Audio settings and I wonder where I can find a preference/settings file on macOS. I cannot find it in ~/Library/... Any hints?

**Comment by ann0see** [comment]
I‘d maybe start a search with spotlight for Jamulus.ini

**Reply by jregener62** [reply]
I have searched for jamulus.* on the entire internal disk, but I haven't found any *.ini or *.pref which might be related to Jamulus. In addition I searched for strings which might come from the Jamulus settings, e.g. my profile name. No success at all.

**Reply by henkdegroot** [reply]
The default filename is Jamulus.ini (and you may have a Jamulusserver.ini as well). I do not have a macOS. Your profile name is not stored in regular text, so it is making sense you could not find that. Try to search for this string: <newclientlevel>
There is also a command line option available which allows you to specify the location/filename to use. The option is --inifile (or -i).
Hope this helps.

**Reply by ghost** [reply]
> I have searched for jamulus.* on the entire internal disk

Maybe the search skips over .hidden directories.
Hidden files are prefixed by a . period.

**Comment by ann0see** [comment]
I think once we found out where the files are saved, it makes sense to document it. Opened: https://github.com/jamulussoftware/jamuluswebsite/issues/588

**Reply by henkdegroot** [reply]
While I do not have a real mac...I do have virtual mac. On this machine I located the Jamulus.ini at the following path:

/Users/user/.config/Jamulus

This contains both the Jamulus.ini and Jamulusserver.ini. Note the "user" in the above is the name of the user.

**Reply by ghost** [reply]
That is similar to linux:
/home/user/.config/Jamulus
(easy to remember)

**Reply by ann0see** [reply]
Interesting. Thanks!

**Comment by chrisrimple** [comment, marked as answer]
From my [Remote Band Rehearsals](https://docs.google.com/document/d/1smcvsxdaaViPQvGMQHmah_6BQeqowhmGSFMHfnlY2FI/) doc...

The INI file can be found:
On a Mac at ~/.config/Jamulus
On Windows at C:\Users\yourusername\AppData\Roaming\Jamulus

**Reply by jregener62** [reply]
Finally got it at ~/.config/Jamulus. Thanks for your hints.
